kentico.gitignore

```diff
@@ -270,7 +270,7 @@ $RECYCLE.BIN/
 # Kentico temporary/environment files
 CMS/App_Data/AzureCache
 CMS/App_Data/AzureTemp
-CMS/App_Data/CMSModules/DeviceProfile/logFiftyOne.txt
+CMS/App_Data/CMSModules/DeviceProfiles/logFiftyOne.txt
```

Comment:
Can you modify this to simply add another line ignoring the plural version for version 10?